src/components/PastTripList.js

```diff
@@ -40,7 +45,9 @@ export default function PastTripList({ notes, onHandleNewNote, onDeleteNote }) {
           />
         </>
       )}
-      {!isActive && <PastTripNotes notes={notes} onDeleteNote={onDeleteNote} />}
+      {!isActive && (
+        <PastTripNotes notes={notes} _id={_id} onDelete={onDelete} />
```

Comment:
You are passing the `_id` as prop but you never receive it. Why do you need the `_id` as prop?